### PR TITLE
Feature: Add ippon ascii banner and api endpoint associated

### PR DESCRIPTION
**Pull request by matthieulapatate**
fix https://github.com/jhipster/jhipster-lite/issues/658

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
- you need to sign the CLA
- in the description of this pull request, you need to add `fix <url of the ticket>`, so the ticket will be closed automatically when this PR is merged

File: src/main/java/tech/jhipster/lite/generator/server/springboot/banner/application/BannerApplicationService.java [hunk 1]
```diff
@@ -17,6 +17,10 @@ public void addBannerJHipsterV7(Project project) {
     bannerService.addBannerJHipsterV7(project);
   }
 
+  public void addBannerJHipsterV7Ippon(Project project) {
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/banner/application/BannerApplicationService.java:20]: nothing related between JHipster and Ippon, so I would rename it to: `addBannerIppon` or `addIpponBanner`

File: src/main/java/tech/jhipster/lite/generator/server/springboot/banner/domain/BannerDomainService.java [hunk 2]
```diff
@@ -20,6 +20,11 @@ public void addBannerJHipsterV7(Project project) {
     addBanner(project, "banner-jhipster-v7.txt");
   }
 
+  @Override
+  public void addBannerJHipsterV7Ippon(Project project) {
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/banner/domain/BannerDomainService.java:24]: same, need to be renamed

File: src/main/java/tech/jhipster/lite/generator/server/springboot/banner/domain/BannerDomainService.java [hunk 3]
```diff
@@ -20,6 +20,11 @@ public void addBannerJHipsterV7(Project project) {
...
+    addBanner(project, "banner-jhipster-v7-ippon.txt");
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/banner/domain/BannerDomainService.java:25]: same, the file should be `banner-ippon.txt`

File: src/main/java/tech/jhipster/lite/generator/server/springboot/banner/infrastructure/primary/rest/BannerResource.java [hunk 4]
```diff
@@ -32,6 +32,15 @@ public void addBannerJHipsterV7(@RequestBody ProjectDTO projectDTO) {
     bannerApplicationService.addBannerJHipsterV7(project);
   }
 
+  @Operation(summary = "Add banner JHipster v7 for Ippon")
+  @ApiResponse(responseCode = "500", description = "An error occurred while adding banner")
+  @PostMapping("/jhipster-v7-ippon")
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/banner/infrastructure/primary/rest/BannerResource.java:37]: just `/ippon` would be enough

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

File: src/main/resources/generator/server/springboot/banner/banner-jhipster-v7-ippon.txt [hunk 5]
```diff
@@ -0,0 +1,26 @@
+
+${AnsiColor.BLUE}          _____          ${AnsiColor.GREEN}          _____          ${AnsiColor.CYAN}          _____         ${AnsiColor.RED}         ______         ${AnsiColor.YELLOW}           _____                                                                                                                 /\    \        ${AnsiColor.YELLOW}           _____
```
Comment (pascalgrimaud) [inline src/main/resources/generator/server/springboot/banner/banner-jhipster-v7-ippon.txt:2]: strange issue at the end of this line:

image

**Comment by pascalgrimaud** [comment]
You miss my comment:

> in the description of this pull request, you need to add fix <url of the ticket>, so the ticket will be closed automatically when this PR is merged

Once the description is updated, I'll merge it :)

**Review by pascalgrimaud — APPROVED** [review]

**Comment by matthieulapatate** [comment]
Note: I dont' know how to change the description once the PR has been created, the edit button only allows to change the title.

**Comment by pascalgrimaud** [comment]
Here, I think:

image

**Comment by matthieulapatate** [comment]
Changed this first comment . Hope this gonna close the issue 😌️

**Comment by pascalgrimaud** [comment]
Thanks and congrats for your 1st contribution @matthieulapatate !

**Pull request merged**